tests/trace_server/test_validation.py

```diff
@@ -42,7 +42,7 @@ def test_validate_purge_req_one():
     assert str(e.value) == "Expected value of type <class 'tuple'>, got <class 'dict'>"
 
     with pytest.raises(InvalidRequest) as e:
-        validation.validate_purge_req_one({"eq_": tuple([{"get_field_": "id"}])})
+        validation.validate_purge_req_one({"eq_": ({"get_field_": "id"},)})
```

Comment:
This is a case where I feel like the other syntax is more readable, but not worth putting in an exception for it.